PUBSTWO-1741 (Crossref ORCID is wrong)

orosz-usgs opened the pull request:
Updated body.ftlx to add the full orcid to the person contributors
  Updated test author contributor to have orcid set.
  src/test/java/gov/usgs/cida/pubs/domain/PublicationContributorHelper.java
  Updated test result crossref xml files to have <ORCID> tag.
  Added testResult/goodCrossref.xml

Commit:
PUBSTWO-1714 (Crossref ORCID is wrong)
  Updated body.ftlx to add the full orcid to the person contributors
  Updated test author contributor to have orcid set.
  src/test/java/gov/usgs/cida/pubs/domain/PublicationContributorHelper.java
  Updated test result crossref xml files to have <ORCID> tag.
  Added testResult/goodCrossref.xml

dsteinich commented on src/main/resources/templates/crossref/body.ftlx:
```diff
@@ -56,7 +56,7 @@
 									</#if>
 									<#if pubContributor.contributor.orcid??>
-										<ORCID>${pubContributor.contributor.orcid}</ORCID>
+										<ORCID>https://orcid.org/${pubContributor.contributor.orcid}</ORCID>
```
Can we use DataNormalizationUtils.ORCID_PREFIX or ${pubContributor.contributor.denormalizeOrcid} instead of hardcoding the url in a second place?

Commit:
PUBSTWO-1741 (Crossref ORCID is wrong)
  Updated body.ftlx to ${ORCID_PREFIX} from the model rather than hard coding
the orcid prefix.
  Updated CrossrefTransformer to add DataNormalizationUtils.ORCID_PREFIX to
the transform model.
